### ISSUE TITLE
Main menu accessibility

### ISSUE THREAD

**Issue by fschroiff** [fronteers/website#508]
**Describe the bug**
[Slack discussion](https://fronteersnl.slack.com/archives/C6MGD72ER/p1737980128349349)

The main menu implements a creative solution for keyboard accessibility. It is not very discoverable though and inaccessible for screen reader users. 

**To Reproduce**
1. Go to fronteers.nl on desktop
2. Tab to the main menu
3. Using left and right arrow keys you can jump between main menu items
4. Using up and down arrow keys you can go into the sub-menus

**Expected behavior**
Good examples for accessible keyboard nav with drop downs: 
https://a11y-collective.github.io/demos/en/advanced-accessible-components/components/navigation/navigation.html
https://www.eur.nl/

- everything accessible by tab
- the "open dropdown" action is triggered by activating a button
- you can tab through the open submenu
- tabbing out of the submenu closes it (tab & shift-tab)
- esc also closes the open submenu
- the submenu trigger has an `aria-label` of "Submenu of [xyz]" and an `aria-expanded` attribute
